src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellItemRenderer.cs

```diff
@@ -84,7 +85,21 @@ public override AView OnCreateView(LayoutInflater inflater, ViewGroup container,
 
 			HookEvents(ShellItem);
 			SetupMenu();
-
+			var appearance = ShellContext.Shell.GetAppearanceForPivot(ShellItem);
+			if (_bottomView.Background is ColorDrawable background)
+			{
+				if (appearance is IShellAppearanceElement appearanceElement)
```

Comment:
Seems that, can combine these two ifs:

```
if (_bottomView.Background is ColorDrawable background 
    && appearance is IShellAppearanceElement appearanceElement)
```

Simplify it a little bit and increase the readability.

Reply:
> Seems that, can combine these two ifs:
> 
> ```
> if (_bottomView.Background is ColorDrawable background 
>     && appearance is IShellAppearanceElement appearanceElement)
> ```
> 
> Simplify it a little bit and increase the readability.

I have simplified the code as per your suggestion. Could you please review it and let me know if you have any concerns?